karma.conf.js

```diff
@@ -14,14 +14,26 @@ const aliases = {
 
 webpackConfig.resolve.alias = Object.assign(webpackConfig.resolve.alias || {}, aliases);
 
+webpackConfig.module.rules = [{
+    test: /\.js$/,
+    include: /(src)\/(js)\//,
+    loader: 'istanbul-instrumenter-loader'
+}];
+
+webpackConfig.module.postLoaders = [{
+    test: /\.js$/,
+    include: /(src)\/(js)\//,
+    exclude: /(test|node_modules)\//,
+    loader: 'istanbul-instrumenter-loader'
+}];
```

Comment:
These config options are not compatible with Webpack 2. 

https://jenkins.jwplayer.com/job/jw7-opensource/3548/console

See the new config options in master and check with @johnBartos on where these loaders should be added.